content/_ext/activenav.rb

```diff
@@ -4,8 +4,17 @@ def active_link?(section_name, options={:css_class => 'active'})
     return options[:css_class] if page.section == section_name
   end
 
+  def absolute_link(relative_url)
+    link =  [site.base_url, relative_url.sub(/^\//, '')].join('/')
+    link.gsub(/\/index.html$/, '/').gsub(/\/(\/)+/, '/')
+  end
+
   def expand_link(relative_url)
-    return [site.base_url, relative_url.sub(/^\//, '')].join('/')
+    link = [URI(site.base_url).path, relative_url.sub(/^\//, '')].join('/')
+    # if it has a file extension its a file and shouldn't get a / added
+    link = link + '/' if File.extname(link).empty?
```

Comment:
This broke all links to anchors, as `/` is also added to those. See e.g. "reporting" labeled link in http://localhost:4242/security/advisory/2021-04-07/#credit